src/components/settingsModal.html

```diff
@@ -0,0 +1,38 @@
+<div class="modal noselect fade" id="settingsModal" tabindex="-1" role="dialog" aria-labelledby="settingsModalLabel">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header" style='justify-content: space-around;'>
+                <span>Settings</span>
+            </div>
+            <div class="modal-body">
+                <div class="row">
+                    <div class="col-md-4">
+                        <div data-bind="template: { name: 'MultipleChoiceSettingTemplate', data: Settings.getSetting('theme')}"></div>
+                    </div>
+                    <div class="col-md-8">
+                        <div data-bind="template: { name: 'MultipleChoiceSettingTemplate', data: Settings.getSetting('test')}"></div>
+                    </div>
```

Comment:
Do you think this should be done via a `ko foreach:Settings.list` thing, and the Settings object can have a property for the template it should use?

Reply:
I've considered that, but then you can't manually order it however you want.

Reply:
True, I guess it would need some kind of `order` property too. Seems like a bit too much UI work for just the couple of settings we have right now. Might be worth revisiting when there are more settings and that html file is growing out of control.